### PR TITLE
Update Drupal 8 References

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes # Multiple tickets in Jira: [DOC-519](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-519), [DOC-520](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-520), [DOC-521](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-521), [DOC-523](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-523), [DOC-532](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-532), [DOC-531](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-531), [DOC-529](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-529), [DOC-528](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-528), [DOC-527](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-527), [DOC-526](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-526), [DOC-525](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-525), [DOC-524](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-524)

## Summary

**[Drupal 8 Performance and Global CDN Caching Settings](https://pantheon.io/docs/drupal-8-cache)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7268](https://pr-7268--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `49` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7268` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/36017/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/36017/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7268--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview: Drupal 9 Cache](https://pr-7268--pantheon-docs.my.pantheonfrontend.website/docs/drupal-9-cache)

File: source/content/core-updates.md [hunk 1]
```diff
@@ -11,15 +11,19 @@ This doc includes instructions to make core updates to WordPress and Drupal site
...
+To check for available updates:
+
+1. Navigate to **Code** in the Dev tab of the site's Dashboard. 
+
+1. Click **Check Now** and then click **Apply Updates** if updates are available.
 
 ## Drupal 8 Composer-Managed Sites
 
-Drupal 8 sites managing core with Composer are not compatible with Pantheon's One-click updates and must update core using Composer exclusively. For instructions, see [Build Tools](/guides/build-tools/update) or [convert the site to Integrated Composer](/guides/composer-convert).
+Drupal 8 sites managing core with Composer are not compatible with Pantheon's One-click updates. You will need to convert to [Drupal 9](/drupal-9) which uses [Integrated Composer](/guides/integrated-composer).
```
Comment (greg-1-anderson) [inline source/content/core-updates.md:22]: It is possible to convert to Integrated Composer without upgrading to Drupal 9:

https://pr-6221--pantheon-docs.my.pantheonfrontend.website/docs/guides/composer-convert

The reference to Build Tools is still relevant, as sites that are still on Build Tools can't use one-click updates.

File: source/content/drupal-9-cache.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Drupal 8 Performance and Global CDN Caching Settings
-description: Optimize Drupal 8 and Glocal CDN caching to maximize your Pantheon site's performance.
+title: Drupal 9 Performance and Global CDN Caching Settings
```
Comment (greg-1-anderson) [inline source/content/drupal-9-cache.md:2]: Will the old content/drupal-8-cache path redirect to drupal-9-cache, or 404?

File: source/content/drupal-9-configuration-management.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ reviewed: "2020-10-21"
 
 Managing configuration is an extremely important part of any team website project, but in many cases, this area of the project does not receive as much attention as it deserves. The tools for Drupal 7 do not provide complete coverage of all configuration settings, leading to inconsistencies in configuration handling and inconvenient workarounds. This has led to configuration management becoming a real thorn in the side for many projects.
 
-Pantheon supports the [Drupal 8 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal site.
+Pantheon supports the [Drupal 9 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal site.
```
Comment (greg-1-anderson) [inline source/content/drupal-9-configuration-management.md:12]: The configuration directory defaults to `/config` for Integrated Composer sites (outside the docroot).

File: source/content/drupal-9-cache.md (hunk 2 above)
Comment (whitneymeredith) [reply source/content/drupal-9-cache.md]: It will redirect.

**Pull request closed without merging**